community/cypher/interpreted-runtime/src/main/scala/org/neo4j/cypher/internal/runtime/interpreted/commands/expressions/PointFunction.scala

```diff
@@ -38,6 +38,7 @@ case class PointFunction(data: Expression) extends NullInNullOutExpression(data)
       if (containsNull(map)) {
         Values.NO_VALUE
       } else {
+        //TODO: We might consider removing this code if the PointBuilder.allowOpenMaps=true remains default
```

Comment:
I think we should indeed remove it, not just consider doing so. It's easy enough to bring it back. And the filtering takes additional time, which is something we can and should avoid.

Reply:
Agreed, but did not want to make it part of this PR this close to release. We wanted the least intrusive change, hence the TODO instead of actually doing it.